### DISCUSSION TITLE
Grid returned by https://api.weather.gov/points/{point} off for points VERY close to polygon border

### DISCUSSION THREAD

**Discussion by dangelsaurus** [weather-gov/api#635] [Q&A]
Found an odd scenario where the grid returned by https://api.weather.gov/points/43.1969,-90.4399 actually has the point 1.1 meters outside of the polygon. (calculated using [ST_Distance](https://postgis.net/docs/ST_Distance.html))  There are others as well.

Any insight into how the points endpoint calculates the appropriate grid? I know it's probably a bit pedantic, but I'm using [ST_Covers](https://postgis.net/docs/ST_Covers.html) to try to find existing cached polygons to cut down on my API calls and these are throwing me curve balls :-)

Thought if I had some insight to how things work behind the scenes, I might be able to program to compensate for these situations better.

**Comment by jah1007** [comment]
Muscoda, WI sits on the edge of the La Crosse, WI forecast office area of responsibility. The next gridpoint to the east and south are covered by the Milwaukee forecast office. I would expect this behavior as it will use the closest gridpoint for the office to the point.
